Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1739,13 +1752,7 @@ void TextureCacheBase::CopyRenderTargetToTexture(u32 dstAddr, EFBCopyFormat dstF
       entry->may_have_overlapping_textures = false;
       entry->is_custom_tex = false;
 
-      // For XFB, the resulting XFB copy texture is the height of the actual texture + the y-scaling
-      // however, the actual Wii/GC texture that we want to "stretch" to this copy is without the
-      // y-scaling so we need to remove it here
-      // The best use-case for this is PAL games which have a different aspect ratio
-      EFBRectangle unscaled_rect = srcRect;
-      unscaled_rect.bottom /= y_scale;
```

Comment:
Note: This here is a big behavior change.

Reply:
Just to let any future readers know, we do still do this for XFBs copied to the host memory.